pkgs/applications/networking/browsers/ephemeral/default.nix

```diff
@@ -0,0 +1,62 @@
+{ stdenv
+, fetchFromGitHub
+, desktop-file-utils
+, gettext
+, glib
+, gtk3
+, hicolor-icon-theme
+, libgee
+, libdazzle
+, meson
+, ninja
+, pantheon
+, pkgconfig
+, python3
+, webkitgtk
+, wrapGAppsHook }:
+
+stdenv.mkDerivation rec {
+  pname = "ephemeral";
+  version = "5.1.0";
+
+  src = fetchFromGitHub {
+    owner = "cassidyjames";
+    repo = "ephemeral";
+    rev = version;
+    sha256 = "1wfrbbdw429q2mkycn87fhci0jidcsflk5f2lbzfzccbcs8msffz";
+  };
+
+  nativeBuildInputs = [
+    desktop-file-utils
+    gettext
+    hicolor-icon-theme
+    libgee
```

Comment:
the final executable links against libgee so I imagine it is a buildInput rather than a nativeBuildInput.

Reply:
yeah this expression needs to be identical to https://github.com/NixOS/nixpkgs/pull/57246/files but with `python3` as native.

Reply:
I change it as @worldofpeace said, but even though, the https problem seems to remain. 

ps: it seems that other commit that I pulled from upstream on my fork have been pushed with my PR, any tips on how I can remove them to clean the PR ?